code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -306,132 +308,202 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
+				"Treadhead" = image('icons/mob/robots.dmi', "Miner"),
+				"Standard" = image('icons/mob/robots.dmi', "Standard-Mine"),
+				"Noble-DIG" = image('icons/mob/robots.dmi', "Noble-DIG"),
+				"Cricket" = image('icons/mob/robots.dmi', "Cricket-MINE"),
+				"Lavaland" = image('icons/mob/robots.dmi', "lavaland")
+			)
+		if("Service")
+			module_sprites = list(
+				"Waitress" = image('icons/mob/robots.dmi', "Service"),
+				"Kent" = image('icons/mob/robots.dmi', "toiletbot"),
+				"Bro" = image('icons/mob/robots.dmi', "Brobot"),
+				"Rich" = image('icons/mob/robots.dmi', "maximillion"),
+				"Default" = image('icons/mob/robots.dmi', "Service2"),
+				"Standard" = image('icons/mob/robots.dmi', "Standard-Serv"),
+				"Noble-SRV" = image('icons/mob/robots.dmi', "Noble-SRV"),
+				"Cricket" = image('icons/mob/robots.dmi', "Cricket-SERV")
+			)
+		if("Combat")
+			module_sprites = list(
+				"Combat" = image('icons/mob/robots.dmi', "ertgamma")
+			)
 		if("Security")
-			module = new /obj/item/robot_module/security(src)
-			module.channels = list("Security" = 1)
-			module_sprites["Basic"] = "secborg"
-			module_sprites["Red Knight"] = "Security"
-			module_sprites["Black Knight"] = "securityrobot"
-			module_sprites["Bloodhound"] = "bloodhound"
-			module_sprites["Standard"] = "Standard-Secy"
-			module_sprites["Noble-SEC"] = "Noble-SEC"
-			module_sprites["Cricket"] = "Cricket-SEC"
-			status_flags &= ~CANPUSH
-
+			module_sprites = list(
+				"Basic" = image('icons/mob/robots.dmi', "secborg"),
+				"Red Knight" = image('icons/mob/robots.dmi', "Security"),
+				"Black Knight" = image('icons/mob/robots.dmi', "securityrobot"),
+				"Bloodhound" = image('icons/mob/robots.dmi', "bloodhound"),
+				"Standard" = image('icons/mob/robots.dmi', "Standard-Secy"),
+				"Noble-SEC" = image('icons/mob/robots.dmi', "Noble-SEC"),
+				"Cricket" = image('icons/mob/robots.dmi', "Cricket-SEC")
+			)
+		if("Destroyer") //for Adminbus presumably
+			module_sprites = list(
+				"Destroyer" = image('icons/mob/robots.dmi', "droidcombat")
+			)
+		if("Hunter")
+			module_sprites = list(
+				"Xeno-Hu" = image('icons/mob/robots.dmi', "xenoborg-state-a")
+			)
+
+	if(custom_sprite && check_sprite("[ckey]-[selected_module]"))
+		module_sprites["Custom"] = image('icons/mob/custom_synthetic/custom-synthetic.dmi', "[ckey]-[selected_module]")
+
+	return module_sprites
+
+/**
+  * Sets up the module items and sprites for the cyborg module chosen in `pick_module()`.
+  *
+  * Arguments:
+  * * selected_module - The name of the module chosen by the player in the previous procs.
+  * * selected_sprite - The name of the sprite chosen by the player in the previous procs.
+  * * module_sprites - The list of sprites possible for the given module. Used to transfer the `icon` and `icon_state` variables to the player.
+  */
+/mob/living/silicon/robot/proc/initialise_module(selected_module, selected_sprite, list/module_sprites)
```

Comment:
this should probably use `initialize` instead of `initialise`

Reply:
i just changed the word, this won't require any more edits to make initialize work, will it? Just checking :)